README.md

```diff
@@ -4,8 +4,11 @@ NASA’s [Open-Source Science Initiative \(OSSI\)](https://science.nasa.gov/open
 
 As part of the OSSI, [SMD’s Scientific Information Policy](https://science.nasa.gov/researchers/science-data/science-information-policy) provides guidance on the open sharing of publications, data, and software created in the pursuit of scientific knowledge. [SPD-41a: Scientific Information Policy for the Science Mission Directorate](https://science.nasa.gov/spd-41/) updates the previously released [SPD-41](https://smd-cms.nasa.gov/wp-content/uploads/2023/05/ScientificInformationpolicySPD_41.pdf), which consolidated existing Federal and NASA policy on sharing scientific information.  SPD-41a was developed in light of new federal guidance on [Ensuring Free, Immediate, and Equitable Access to Federally Funded Research](https://www.whitehouse.gov/wp-content/uploads/2022/08/08-2022-OSTP-Public-Access-Memo.pdf), studies from the National Academies, and input from the SMD scientific community. 
 
-This repository provides guidelines, best practices, and examples of open-source science to support the SMD scientific community in implementing the requirements of SPD-41a and achieving the broad OSSI goal of moving science towards openness. **Please note the [Limitations](#limitations) to this guidance.** This guidance represents general recommendations and best practices for the SMD scientific community and should be considered alongside any additional guidance provided by SMD Divisions or specific funding solicitations. For additional information about SPD-41a and other scientific information policies that correspond to SMD Divisions, see the [Scientific Information Policy website](https://science.nasa.gov/researchers/science-data/science-information-policy) and [Frequently Asked Questions \(FAQ\)](https://science.nasa.gov/researchers/science-data/science-information-policy_faq). For additional open science resources and training opportunities, see NASA’s see [NASA’s Transform to Open Science \(TOPS\) program](https://nasa.github.io/Transform-to-Open-Science/). 
+This repository provides guidelines, best practices, and examples of open-source science to support the SMD scientific community in implementing the requirements of SPD-41a and achieving the broad OSSI goal of moving science towards openness. **Please note the [Limitations](#limitations) to this guidance.** This guidance represents general recommendations and best practices for the SMD scientific community and should be considered alongside any additional guidance provided by SMD Divisions or specific funding solicitations. For additional information about SPD-41a and other scientific information policies that correspond to SMD Divisions, see the [Scientific Information Policy website](https://science.nasa.gov/researchers/science-data/science-information-policy) and [Frequently Asked Questions \(FAQ\)](https://science.nasa.gov/researchers/science-data/science-information-policy_faq). 
```

Comment:
Suggestion add the OSDMP page as well: https://science.nasa.gov/researchers/sara/faqs/osdmp/

Reply:
Added.